ui-react-starter/src/components/About.jsx

```diff
@@ -5,33 +5,31 @@ import os from "os";
-                    This is a simple plugin that demonstrates the various capabilities of React on UXP.
-                    When adapting to your own projects, you can replace <code>index.jsx</code> and the components
-                    with your own. 
-                </sp-body>
-                <sp-body class="well">
-                    <sp-icon name="ui:InfoSmall" size="s"></sp-icon>
-                    We've also included the `WC` component and a couple of controlelrs. You
-                    do not need to use these in your own projects, but you are welcome to do so.
-                </sp-body>
-                <sp-detail>VERSIONS</sp-detail>
-                <div className="table">
-                    <div><sp-detail>PLUGIN: </sp-detail><sp-body> {versions.plugin}</sp-body></div>
-                    <div><sp-detail>OPERATING SYTEM:</sp-detail><sp-body> {os.platform()} {os.release()}</sp-body></div>
-                    <div><sp-detail>UNIFIED EXTENSIBILITY PLATFORM:</sp-detail><sp-body>{versions.uxp}</sp-body></div>
-                </div>
-                <sp-button-group>
-                    <sp-button tabindex={0} variant="secondary" quiet="quiet" onClick={() => this.props.dialog.close("reasonCanceled")}>Cancel</sp-button>
-                    <sp-button tabindex={0} autofocus="autofocus" variant="primary" onClick={() => this.props.dialog.close("ok")}>Ok</sp-button>
-                </sp-button-group>
-            </form>
-        );
-    }
+export const About = () => {
+    return (
+        <form method="dialog" className="aboutDialog">
+        <sp-heading>React Starter Plugin</sp-heading>
+        <sp-divider size="large"></sp-divider>
+        <sp-body>
+            This is a simple plugin that demonstrates the various capabilities of React on UXP.
+            When adapting to your own projects, you can replace <code>index.jsx</code> and the components
+            with your own. 
+        </sp-body>
+        <sp-body class="well">
+            <sp-icon name="ui:InfoSmall" size="s"></sp-icon>
+            We've also included the `WC` component and a couple of controlelrs. You
+            do not need to use these in your own projects, but you are welcome to do so.
+        </sp-body>
+        <sp-detail>VERSIONS</sp-detail>
+        <div className="table">
+            <div><sp-detail>PLUGIN: </sp-detail><sp-body> {versions.plugin}</sp-body></div>
+            <div><sp-detail>OPERATING SYTEM:</sp-detail><sp-body> {os.platform()} {os.release()}</sp-body></div>
+            <div><sp-detail>UNIFIED EXTENSIBILITY PLATFORM:</sp-detail><sp-body>{versions.uxp}</sp-body></div>
+        </div>
+        <sp-button-group>
+            <sp-button tabindex={0} variant="secondary" quiet="quiet" onClick={() => this.props.dialog.close("reasonCanceled")}>Cancel</sp-button>
+            <sp-button tabindex={0} autofocus="autofocus" variant="primary" onClick={() => this.props.dialog.close("ok")}>Ok</sp-button>
+        </sp-button-group>
```

Comment:
A couple of things here:

1. Can you make sure the dialog closes when these buttons are clicked? Wild guess, but it might be worth checking if `.dialog` is being passed in `this.props`.
2. Minor nit: Change "Ok" to "OK"

These may have been issues in the original sample as well, and this is a good time to fix them.

Reply:
> These may have been issues in the original sample as well, and this is a good time to fix them.

Well, typos were present, but the buttons were not working due to something I messed up (didn't pass props to the functional component). :p
This should be fixed now.

Reply:
Nice!